pulumi/grapl/__main__.py

```diff
@@ -125,6 +135,30 @@ def main() -> None:
 
         kafka = Kafka("kafka")
 
+        job_vars = pulumi.Output.all(
+            graph_merger_queue=graph_merger_queue.main_queue_url,
+            graph_merger_dead_letter_queue=graph_merger_queue.dead_letter_queue_url,
+            session_table_name=dynamodb_tables.dynamic_session_table.name,
+            schema_properties_table_name=dynamodb_tables.schema_properties_table.name,
+            schema_table_name=dynamodb_tables.schema_table.name,
+            node_identifier_queue=node_identifier_queue.main_queue_url,
+            node_identifier_dead_letter_queue=node_identifier_queue.dead_letter_queue_url,
+            node_identifier_retry_queue=node_identifier_queue.retry_queue_url,
+            subgraphs_merged_bucket=subgraphs_merged_emitter.bucket,
+            subgraphs_generated_bucket=subgraphs_generated_emitter.bucket,
+            user_auth_table=dynamodb_tables.user_auth_table.name,
+            ux_bucket=ux_bucket.bucket,
+        ).apply(
+            lambda inputs: {
+                "deployment_name": DEPLOYMENT_NAME,
+                "grapl_test_user_name": f"{DEPLOYMENT_NAME}-grapl-test-user",
```

Comment:
Looks like we can just use GRAPL_TEST_USER_NAME

Reply:
I've removed the deployment_name since you're correct that its unnecessary for nomad

Reply:
oh no I mean theres literally a constant in pulumi called GRAPL_TEST_USER_NAME